src/View/User/show.html.twig

```diff
@@ -0,0 +1,84 @@
+{% extends 'layout.html.twig' %}
+
+{% block stylesheet %}
+  {{ parent() }}
+  <link rel="stylesheet" href="/assets/css/show.css">
+  <link rel="stylesheet" href="/assets/css/userForm.css">
+{% endblock %}
+
+{% block content %}
+
+<aside id="id01" class="modal">
+  	<span onclick="document.getElementById('id01').style.display='none'" class="close" title="Close Modal">×</span>
+  	<section class="formSection formSectionModale">
+ 
+      <h1>Delete Account</h1>
```

Comment:
only one h1 by page, and not really optimized to focus the main title of the page for a deletion title